### PR TITLE
Add tests for firstOrCreate

### PR DESCRIPTION
**Pull request by bolechen**

### CODE REVIEW

File: tests/Features/Methods/ModelExtension.php [hunk 1]
```diff
@@ -141,6 +141,11 @@ public function testUpdateOrCreate() : User
         return User::updateOrCreate([]);
     }
 
+    public function testfirstOrCreate() : User
```
Comment (canvural) [inline tests/Features/Methods/ModelExtension.php]: Can you change this to `testFirstOrCreate` ? To follow the same style as others.

Comment (bolechen) [reply tests/Features/Methods/ModelExtension.php]: ok, sorry newbie for github.

Comment (bolechen) [reply tests/Features/Methods/ModelExtension.php]: is now ok?

Comment (canvural) [reply tests/Features/Methods/ModelExtension.php]: No, still the same.

First update your fork with the latest changes of the `master` branch.

Comment (canvural) [reply tests/Features/Methods/ModelExtension.php]: If you have `upstream` remote defined in your fork, you can do `git pull upstream master` to update your fork.

Comment (bolechen) [reply tests/Features/Methods/ModelExtension.php]: ok, let me try.

**Pull request merged**